client/app/src/main/java/com/microsoft/research/karya/data/service/KaryaFileAPI.kt

```diff
@@ -0,0 +1,25 @@
+package com.microsoft.research.karya.data.service
+
+import com.google.gson.JsonObject
+import okhttp3.MultipartBody
+import okhttp3.ResponseBody
+import retrofit2.Response
+import retrofit2.http.*
+
+interface KaryaFileAPI {
+
+    @Multipart
+    @POST("/karya_files")
+    suspend fun uploadKaryaFile(
+        @Header("id-token") idToken: String,
+        @Part json: MultipartBody.Part,
+        @Part file: MultipartBody.Part
+    ): Response<JsonObject>
```

Comment:
Can you create a model for the response instead of using `JsonObject`

Reply:
Sure